File: 250721/점수 비교/score-comparison.py
# 입력
intAmath, intAeng = map(int, input().split())
intBmath, intBeng = map(int, input().split())

# 조건문
if (intAmath > intBmath) and (intAeng > intBeng) :
    print(1)
else :
    print(0)

File: 250721/점수 비교/score-comparison.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intAmath = sc.nextInt();
        int intAeng = sc.nextInt();
        int intBmath = sc.nextInt();
        int intBeng = sc.nextInt();

        // 조건문
        if ((intAmath > intBmath) && (intAeng > intBeng)) {
            System.out.println(1);
        } else {
            System.out.println(0);
        }
    }
}